roles/kubernetes/preinstall/tasks/main.yml

```diff
@@ -97,6 +97,14 @@
     - canal
     - bootstrap-os
 
+- include: resolvconf.yml
+  when:
+    - dns_mode != 'none'
+    - resolvconf_mode == 'host_resolvconf'
+  tags:
+    - bootstrap-os
+    - resolvconf
+    
```

Comment:
Please remove trailing space

Reply:
Hello,
Actually, if I remove this space, there will be no separation with  the next task.
Patrick

Reply:
I mean it should be an empty line with no spaces.